_data/reviews.yml

```diff
@@ -189,4 +189,11 @@
         I especially appreciated how she shared her knowledge of real-world systems, offering valuable context and insight.  
         Her support and constructive advice have been instrumental in boosting my confidence and preparation.
       name: Sonali Goel
+      type: Ad-Hoc
+- index: 39 # Busra
+  feedback:
+    - score: 5
```

Comment:
What about the feedback? @mads6879 i guess the description was mandatory in the form could you confirm?